OOBE/6.0.0/member_groups/download_qualities/download_qualities.dcl

```diff
@@ -8,14 +8,6 @@ resource member_group download_qualities {
     roles = []
     download_qualities = [{
             media_format_id = local.source_media_format_magic_id
```

Comment:
```suggestion
    download_qualities = [{
            media_format_id = local.source_media_format_magic_id
        }, {
            media_format_id = data.format.video_preview.id
        }, {
            media_format_id = data.format.large_thumbnail.id
```
I think that we might need to add the appropriate data references as well :)

Reply:
Kasper had already been nice and had done this for 5.10, so just merged master :D